backend/api/Models/Parcel/ParcelModel.cs

```diff
@@ -21,7 +21,7 @@ public class ParcelModel : PropertyModel
 
         public IEnumerable<ParcelFiscalModel> Fiscals { get; set; } = new List<ParcelFiscalModel>();
 
-        public IEnumerable<ParcelBuildingModel> Buildings { get; set; } = new List<ParcelBuildingModel>();
+        public virtual IEnumerable<ParcelBuildingModel> Buildings { get; set; } = new List<ParcelBuildingModel>();
```

Comment:
Is there a reason it's `virtual`?

Reply:
I'll check, I thought I was overriding this somewhere.

Reply:
Yes, overriden in the buildingParcelModel